packages/synapse-react-client/src/components/IconList.tsx

```diff
@@ -29,6 +31,15 @@ function IconList(props: IconListProps) {
   let noMatch: boolean = false
   const css = useTheme ? 'icon-list themed' : 'icon-list'
   const componentCss = useBackground ? `${css} bg-circle` : css
+  const queryContext = useQueryContext()
+  const { addValueToSelectedFacet } = queryContext
+
+  const handleIconClick = (dataType: 'string') => {
```

Comment:
Function is recreated on rerender (I think).  Is this a good place for a useCallback?  (It's unclear to me if IconSvg is going to be re-rendered when this is recreated)